orchard-provider-aws/src/main/scala/com/salesforce/mce/orchard/io/aws/resource/EmrResource.scala

```diff
@@ -242,7 +246,7 @@ object EmrResource {
     .validate[Spec]
     .map { spec =>
       val loggingPath = s"${conf.workflowId}_rsc-${conf.resourceId}_${conf.instanceId}"
-      EmrResource.apply(conf.resourceName, loggingPath, spec)
+      EmrResource.apply(conf.resourceName, loggingPath, spec, conf.instanceId >= conf.maxAttempt)
```

Comment:
@anniezhang77 nvm, I can see the `instanceId` is the `instance_attempt`, e.g. `wf-dfd5dc80-957a-428b-91c4-d480dc429133_rsc-c4c36105-cd36-42cc-8038-691fea220731_1` for loggingPath where `1` is the `instanceId`